src/sentry/db/models/paranoia.py

```diff
@@ -24,7 +24,7 @@ def get_queryset(self) -> ParanoidQuerySet:
         return ParanoidQuerySet(self.model, using=self._db).filter(date_deleted__isnull=True)
 
 
-class ParanoidModel(Model):
+class ParanoidModel(BaseModel):
```

Comment:
To share `ParanoidModel` with `ReplicatedModel`, only one of them can subclass from `Model` or you get field conflicts in the meta class.  Can verify that this is a non impact in practice.

Reply:
Could this switch in base classes impact backup/import as all the hook methods that backup uses are defined there.

Reply:
Read this again, and I get it now.